File: FleemanWithJava/src/main/java/com/example/models/AddonBook.java
package com.example.models;
import jakarta.persistence.*;
@Entity
@Table(name="Add_on_Book")
public class AddonBook {
	@Id
	private Integer bookingId;	
	private  Short AddonId;

	/*Getter-Setters*/
	public Integer getBookingId() {
		return bookingId;
	}

	public void setBookingId(Integer bookingId) {
		this.bookingId = bookingId;
	}

	public Short getAddonId() {
		return AddonId;
	}

	public void setAddonId(Short addonId) {
		AddonId = addonId;
	}

}

File: FleemanWithNET/Models/AddOnBook.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace fleeman_with_dot_net.Models
{
    [Table("add_on_book")]  // your join table name
    public class AddOnBook
    {
        [Key]
        public int Id { get; set; }  // Primary key for this join table

        public int BookingId { get; set; }  // Foreign key to booking

        public int add_on_id { get; set; }  // Foreign key to add-on
    }
}
